clouddrift/select.py

```diff
@@ -1,40 +1,41 @@
-import awkward as ak
+import xarray as xr
 import numpy as np
+import warnings
 
 
-def mask_var(var: ak.Array, value):
+def mask_var(var: xr.DataArray, value):
```

Comment:
I suggest `criteria` (consistent with `subset`), or even better, `condition`:

```suggestion
def mask_var(var: xr.DataArray, criteria):
```

Also, add type hints for arguments and result.